tests/unit/lib/SGVector_unittest.cc

```diff
@@ -181,12 +184,9 @@ TEST(SGVectorTest,complex128_tests)
 	for (index_t i=0; i<res.vlen; ++i)
 		EXPECT_EQ(res[i], i);
 
-	a.scale(complex128_t(1.0));
-	for (index_t i=0; i<a.vlen; ++i)
-	{
-		EXPECT_NEAR(a[i].real(), 10.0, 1E-14);
-		EXPECT_NEAR(a[i].imag(), 12.0, 1E-14);
-	}
+	// removed EXPECT_NEAR block for SFVector::scale
+	// but inserted linalg because further tests are made on data after scaling
+	linalg::scale<linalg::Backend::NATIVE>(a, complex128_t(1.0));
```

Comment:
same as above. Also, not a good idea to remove existing unit-tests. Just imagine, if the scale fails somehow, we'll get to see failed tests with misleading names (i.e. the later tests). Having specific unit-tests for small things serves just that purpose. It helps us to pinpoint the exact cause of a failure.